.github/workflows/ci-core.yml

```diff
@@ -102,6 +114,10 @@ jobs:
       - name: Download Go vendor packages
         if: ${{ needs.filter.outputs.changes == 'true' }}
         run: go mod download
+      - name: Replace chainlink-evm deps
+        if: ${{ inputs.evm-ref != ''}}
```

Comment:
Want to also check for `needs.filter.outputs.changes == 'true'` ?

Reply:
Speaking of, would the `needs` job that checks for changes inadvertently kill this workflow when doing the dispatch trigger?

Reply:
@kalverra I think I started working on this before the filtering was added so I will need to look again. As for the filter itself I don't know if it is setup correctly or not. I haven't used it in a negating way before, will it currently only run if there are changes only outside of the integration tests directory? Will it still run if there are changes in both?

Reply:
@kalverra The lack of the needs for the filter didn't seem to matter and I am not sure why exactly. I put the same input to force the filter to true for workflow dispatch events that I had in the integration-tests workflow so at least the logic makes sense for why it runs now.